docsite/docs/Docs/fal-cli/python-models.md

```diff
@@ -0,0 +1,55 @@
+---
+sidebar_position: 2
+---
+
+# Making Python dbt models
+
+> ⚠️ NOTE: This feature is enabled with the flag `--experimental-models`
+
+There are some cases where a SQL transformation is either too complex or impossible. With `fal flow` you can build your dbt models in Python.
+
+To achieve this, add the Python (`.py`) or [Notebook (`.ipynb`)](./notebook-files.md) files to any of your [`model-paths`](https://docs.getdbt.com/reference/project-configs/model-paths) and make sure your script calls the [`write_to_model`](../../Reference/variables-and-functions.md#writetomodel-function) function _exactly once_.
+
+## Dependencies
+
+`fal` should pick up most usages of [`source`](../../Reference/variables-and-functions.md#source-function) or [`ref`](../../Reference/variables-and-functions.md#ref-function) functions and generate the appropriate dependency for the `dbt` DAG. In case a dependency is not being picked up by `fal`, you can specify them in the top of the Python file in the [module docstring](https://peps.python.org/pep-0257/):
```

Comment:
`fal` will resolve any usage of [`source`](../../Reference/variables-and-functions.md#source-function) or [`ref`](../../Reference/variables-and-functions.md#ref-function) functions and generate the appropriate dependencies for the `dbt` DAG. In case a dependency is not being picked up by `fal`, you can specify it on top of the Python file in the [module docstring](https://peps.python.org/pep-0257/):